gdk4/src/auto/toplevel_layout.rs

```diff
@@ -57,10 +66,11 @@ impl ToplevelLayout {
     }
 
     #[doc(alias = "gdk_toplevel_layout_get_maximized")]
-    pub fn get_maximized(&self) -> bool {
+    pub fn get_maximized(&self, maximize: bool) -> bool {
```

Comment:
This is wrong. `maximize` in the C API is a pointer to a bool, it's an out parameter. Missing annotation

Reply:
Should work the same as `get_fullscreen()` basically

Reply:
Seems to be fixed already in-tree, I have re-updated the gir-files in https://github.com/gtk-rs/gir-files/pull/71